docs/using/gui/index.rst

```diff
@@ -1,11 +1,26 @@
 .. _gui:
```

Comment:
Just as a suggestion, you may want to change this to something like ```.. _widgets:``` so it would be easier for people to work with in the future. I don't remember where this document is hyperlinked though but I think there is at least one spot where it is.

Reply:
@smithis7 Good point - `visualization` will be most suitable!